src/server/index.js

```diff
@@ -18,6 +18,9 @@ app.get('/heartbeat', function(req, res) {
 app.get('/login', (req, res) => {
   res.sendFile(path.resolve(__dirname, '../../dist/index.html'));
 });
+app.get('/register', (req, res) => {
```

Comment:
use the array version of `app.get`
http://expressjs.com/en/api.html#app.get.method